### PR TITLE
maven-publish-plugin adoption

### PR DESCRIPTION
**Pull request by chrisdennis**
This probably isn't good enough to be merged but I think it could do with some review. I'm happy enough with the individual modules but the shading logic bothers me. I'm starting to lean towards replacing the shadow plugin with something homegrown since it seems to be lacking somewhat when it comes to creating partial shades, and interacting nicely with other plugins (e.g. bnd).

### CODE REVIEW

File: clustered/clustered-dist/build.gradle [hunk 1]
```diff
@@ -63,15 +63,15 @@ dependencies {
...
+  shadow "org.slf4j:slf4j-api:$parent.slf4jVersion"
+  shadow project(':dist')
 }
 
-task copyDocs(type: Copy) {
-  dependsOn asciidocZip
-  from zipTree(asciidocZip.archivePath)
-  into docsFolder
-}
+//task copyDocs(type: Copy) {
```
Comment (anthonydahanne) [inline clustered/clustered-dist/build.gradle]: you could delete those comments ?

File: osgi-test/build.gradle [hunk 2]
```diff
@@ -83,6 +84,7 @@ test {
   }
 }.doFirst {
   configurations.osgiModule.resolvedConfiguration.resolvedArtifacts.forEach({
+    println "$it.moduleVersion.id.module:osgi-path -> $it.file"
```
Comment (anthonydahanne) [inline osgi-test/build.gradle]: not needed anymore ?

**Comment by anthonydahanne** [comment]
maybe you shoudl add a specific commit for the new transaction module ?

**Comment by chrisdennis** [comment]
Closing this aborted attempt.

**Pull request closed without merging**